src/main/kotlin/view/OutputView.kt

```diff
@@ -24,17 +27,30 @@ class OutputView {
         println(INPUT_LOTTO_NUMBER_MESSAGE)
     }
 
-    fun printLottoTickets(lottoTickets: List<LottoTicket>) {
-        val manualTickets = lottoTickets.filter { it.type == TicketType.Manual }
-        val autoTickets = lottoTickets.filter { it.type == TicketType.Auto }
-        println("수동으로 ${manualTickets.size}장, 자동으로 ${autoTickets.size}장을 구매하셨습니다.")
-        if (manualTickets.isNotEmpty()) {
+    fun printLottoTickets(lottoTickets: LottoTickets) {
+        printLottoGames(lottoTickets.asLottoGames())
+        println("총 ${lottoTickets.elements.size}장의 티켓을 구매했습니다.")
+    }
+
+    private fun printLottoGames(lottoGames: List<LottoGame>) {
+        val manualGames = lottoGames.filter { it.type == TicketType.Manual }
+        val autoGames = lottoGames.filter { it.type == TicketType.Auto }
+        println("수동으로 ${manualGames.size}게임, 자동으로 ${autoGames.size}게임을 구매하셨습니다.")
```

Comment:
```suggestion
   private fun printLottoGames(lottoGames: List<LottoGame>) {
    val groupedGames = lottoGames.groupBy { it.type }

    groupedGames.forEach { (type, games) ->
        val typeString = if (type == TicketType.Manual) "수동" else "자동"
        println("$typeString 으로 ${games.size} 게임을 구매했습니다.")
        printLottoNumbersForType(games, typeString)
    }
}

private fun printLottoNumbersForType(games: List<LottoGame>, typeString: String) {
    if (games.isNotEmpty()) {
        println("$typeString 으로 구매한 로또 번호")
        games.forEach { println(it.numbers) }
    }
}
```

groupBy 라는 녀석도 있더라구요 재밌는 거 같습니다 ㅎ_ㅎ

Reply:
적절하게 활용한다면 좋을 것 같습니다 ㅎㅎ groupBy 를 쓰면 가독성이 다소 난해해질 수 있을 것 같아요

Reply:
현재 처럼 딱 2개면 좀 애매한 거 같습니다